reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/common/ParseThread.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 Facebook, Inc.
+ * <p>
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *  <p>
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  <p>
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations under the License.
+ */
+
+package io.reactivesocket.tckdrivers.common;
+
+public class ParseThread implements Runnable {
```

Comment:
I'd avoid code that manually creates threads and even implementing runnable.  This class could just be an inline lambda passed to an ExecutorService.